test/tests/misc/app_name/tests

```diff
@@ -3,7 +3,7 @@
     type = 'RunApp'
     cli_args = '--type'
     expect_out = "MooseTestApp"
-    input = 'IGNORED'
+    input = ''
```

Comment:
Why all of the changes from

```
input = 'IGNORED'
```

to

```
input = ''
```

?

Reply:
This is suggested by @permcody.  In MooseApp.C we have a very long `if elseif and else` condition to deal with different command line options. When we want to ignore input files, however we actually don't want to use 'IGNORED' as a key word to detect "ignore input file condition", it just fells into the else condition in the very end. So we decide to replace the 'IGNORED' key word in test with ''. Therefore the logic below can work, otherwise it will looking for input file named 'IGNORED' in the folder.
```
  input_param.addCommandLineParam<std::vector<std::string>>(
      "input_file",
      "-i <input_files>",
      "Specify one or multiple input files. Multiple files get merged into a single simulation "
      "input.");

  command_line->addCommandLineOptionsFromParams(input_param);

  std::vector<std::string> input_filename;
  command_line->search("input_file", input_filename);

  if (!input_filename.empty())
    front_parser->parse(input_filename);
```

Reply:
Works for me.